documentation/bolt_configuration_options.md

```diff
@@ -54,6 +55,22 @@ ssh:
 -   `tty`: Request a pseudo tty for the SSH session. This option is generally only used in conjunction with the `run_as` option when the sudoers policy requires a `tty`. Default is `false`.
 -   `user`: Login user. Default is `root`.
 
+For example:
+
+```yaml
+targets:
+  - name: host1.example.net
+    config:
+      transport: ssh
+      ssh:
+        host-key-check: true
+        port: 22
+        run-as-command: ['sudo', '-k', '-n']
+        private-key:
+          key-data: |
```

Comment:
I had forgotten about this option and realized it wasn't documented. It's a bit hard to describe in words, so I added an example of that as well as the other non-string option `run-as-command` for clarity.